modules/auth/middleware.js

```diff
@@ -0,0 +1,67 @@
+const Joi = require("joi");
+const passport = require("passport");
+const User = require("./model");
+const JwtStrategy = require("passport-jwt").Strategy,
+  ExtractJwt = require("passport-jwt").ExtractJwt;
```

Comment:
nie rob tak. kazdy `const` na oddzielnej linii. (czyli `ExtractJwt` rowniez musi byc zapisany na `const` bezposrednio). To pomaga ustrzec programiste przed bledami i stalo sie standardem pisania kodu. (Twoj team moze miec inny pomysl, ale wtedy dostosujesz sie do teamowych guidelines. Na potrzebe tego kursu, Mentor 'dyktuje' guidelines.)